Adding packer to tyk-build-env

alephnull opened the pull request:
Also noted that build pipelines were using wrong version of tyk-build-env. New version `1.3` pushed and tagged as `latest`. All pipelines should now be using latest tag.

excieve commented:
@alephnull  Could you please elaborate on why is packer needed in this one?

IIRC this image is also the base image for plugin builder. Packer is really only needed to our internal pipelines that build AMIs, no?

alephnull commented:
> Packer is really only needed for our internal pipelines that build AMIs, no?
Yes, correct. 

packer used to be installed via a custom env section in buddy but when I moved everything to use the build-env base image, I removed that section as it was 
- easy to forget that it was there
- updating the packer version was cumbersome

Perhaps I should make another image that can be used for AMI builds.

excieve commented:
@alephnull OK, let's keep it in this image for now — it's silly huge anyway :) Split later if it becomes an issue.

alephnull commented:
> @alephnull OK, let's keep it in this image for now — it's silly huge anyway :)

Agreed. With a little work I might be able to hive off all of ruby from this. python-dev will still keep it large though.
